shell/pages/home.vue

```diff
@@ -301,7 +301,7 @@ export default {
             data-testid="changelog-banner"
             color="info whats-new"
           >
-            <div class="message">
+            <div>
```

Comment:
What do the changes in this file do? These banners seem to fine now in the master branch - are these changes needed?

Reply:
Yes, I added this class in the first PR to fix the home banner and did some changes in the banner component. But after new fixes we don't need these extra classes.